composer.json

```
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": "^7.1.3|^8.0",
```

For the record, my personal preference would be the following constraint.

```suggestion
        "php": ">=7.1.3",
```

But I'd like to leave it up to the maintainers to define the strategy here.